from exc.MyException import CustomException


class ControllerException(CustomException):
    pass